netty/src/main/java/io/grpc/netty/HandlerSettings.java

```diff
@@ -0,0 +1,16 @@
+package io.grpc.netty;
+
+import java.util.ArrayList;
+
+public class HandlerSettings {
```

Comment:
Please add a java doc explaining why this class exists (and hopefully when it can be removed)